### ISSUE TITLE
Perform user study on Prolific

### ISSUE THREAD

**Issue by SebastianZimmeck** [privacy-tech-lab/gpc-privacy-choice#199]
This issue is for planning and discussion purposes. There is nothing concretely to do.

Based on what we discussed in our call today, here is the workflow and an approximate timeline for our study.:

### **Scheme 1**
- Set extension to **scheme 1**, test it, fix if necessary, and create scheme 1 GitHub release (v.2.1.0).
- Deploy v.2.1.0 on the Chrome Web Store

### **Scheme 2**
- Start collecting user data for scheme 1 and in parallel set extension to **scheme 2**, test it, fix if necessary, and create scheme 2 GitHub release (v.2.2.0).
- Deploy v.2.2.0 on the Chrome Web Store
- Start collecting user data for scheme 2

### **Scheme 3**
- Continue collecting user data for open schemes and in parallel set extension to **scheme 3**, test it, fix if necessary, and create scheme 3 GitHub release (v.2.3.0).
- Deploy v.2.3.0 on the Chrome Web Store
- Start collecting user data for scheme 3

### **Scheme 4**
- Continue collecting user data for open schemes and in parallel set extension to **scheme 4**, test it, fix if necessary, and create scheme 4 GitHub release (v.2.4.0).
- Deploy v.2.4.0 on the Chrome Web Store
- Start collecting user data for scheme 4

### **Scheme 5**
- Continue collecting user data for open schemes and in parallel set extension to **scheme 5**, test it, fix if necessary, and create scheme 5 GitHub release (v.2.5.0).
- Deploy v.2.5.0 on the Chrome Web Store
- Start collecting user data for scheme 5

### **Scheme 6**
- Continue collecting user data for open schemes and in parallel set extension to **scheme 6**, test it, fix if necessary, and create scheme 6 GitHub release (v.2.6.0).
- Deploy v.2.6.0 on the Chrome Web Store
- Start collecting user data for scheme 6

### **Scheme 0**
- Continue collecting user data for open schemes and in parallel set extension to **scheme 0**, test it, fix if necessary, and create scheme 0 GitHub release (v.2.7.0).
- Deploy v.2.7.0 on the Chrome Web Store
- Start collecting user data for scheme 0

### Finalization
- Finalize collecting user data..
- Unrelease Chrome extension and update readme accordingly

We can keep track of the dummy Prolific ID we are using for testing to not accidentally delete real user data.

**Comment by SebastianZimmeck** [comment]
I updated the timeline. @bella-tassone is going to take the lead on this. We discuss tomorrow.

**Comment by SebastianZimmeck** [comment]
@bella-tassone, once #218 is fixed, can you prepare a new release? By the end of the week would be good.

Two important points. 
1. The background.js needs to be set to scheme 1. I believe @chunyuema said that that is already the case, but please make sure.
2.  Please update the manifest with "v.2.1.0" (and call the release here on GitHub accordingly); also provide a brief changelog of what is new and fixed; only the top three things or so (you can check the issues to remind yourself).

**Comment by SebastianZimmeck** [comment]
Per #220:

> @bella-tassone, can you go ahead and prepare release v.2.1.0 here on GitHub?

> Yes, I've just been waiting for @ekuller to update the readme! Eliza, do you have any update there?

I am closing #220 as we can have this discussion here for all releases and the full process over the next few weeks.

**Comment by SebastianZimmeck** [comment]
@bella-tassone, can you go ahead with the release? @ekuller can modify the readme later.

**Comment by bella-tassone** [comment]
> @bella-tassone, can you go ahead with the release? @ekuller can modify the readme later. 

I'll do it today! :)

**Comment by SebastianZimmeck** [comment]
I submitted v2.1.0 Revision for review on the Chrome Web Store. I'll provide an update once we have a decision ...

**Comment by SebastianZimmeck** [comment]
This was quick! v2.1.0 Revision already approved on the Chrome Web Store!

**Comment by SebastianZimmeck** [comment]
Two more modifications for the next release:

1. We may want to add a prominent link to the options page for selecting individual domains on schemes 3-6
2. Per @bella-tassone, we’re definitely gonna want to specify the registration tab for later schemes

**Comment by bella-tassone** [comment]
Additionally, we may want to notify users that they have to allow pop-up tabs in order to see the registration!

**Comment by SebastianZimmeck** [comment]
As just discussed, @bella-tassone will double-check whether all schemes have a domain list (we believe they do). Especially,  it is important for scheme 6 Enabling/Disabling GPC for All Sites. So, if scheme 6 does not have the domain list, we should implement it there.

For non-banner schemes, we can include a sentence linking to the domain list. For example, for scheme 4 "Hover over the cards to learn more. You can also <enable or disable GPC for individual sites>." The bracketed language is a link to the domain list.

@bella-tassone, please feel free to convert this comment to an individual issue if you think it is better (which is generally true for everything we discuss in this omnibus issue; if a point is getting to big to continue discussing here, it may make sense to create a separate issue for the comment).

**Comment by bella-tassone** [comment]
@SebastianZimmeck after thinking about it, I don't think it would really make sense to have a link going to the Options page on schemes 3-6. If this link was implemented on the registration page (which we discussed before) then users would **only** see it when they were registering, which would actually be before they had any domains _in_ the domain list. Thus, it would be unusable. Moreover, we need the users to register before they do anything else, and having a link to the Options page on the registration tab would defeat the purpose there.

However, I think it would be beneficial to include maybe a **sentence** in the registration tab for schemes 3-6 that pointed out the Options page, and let users know they could make individual domain decisions there. Let me know how you feel about this compromise!

**Comment by SebastianZimmeck** [comment]
> I think it would be beneficial to include maybe a sentence in the registration tab for schemes 3-6 that pointed out the Options page

@bella-tassone, what do you mean by registration tab? Do you mean the Prolific survey ad?

Also, can you confirm that schemes 3-6 have the domain list already implemented?

**Comment by bella-tassone** [comment]
No, I mean the page that pops up when users first install the browser extension – what we have pictured in the readme for schemes 3-6. And yes, I can confirm the domain list is implemented for all schemes.

**Comment by bella-tassone** [comment]
> OK, sounds good! Let's do that then.

I'll get right on it!
